pkg/util/output.go

```diff
@@ -24,9 +24,15 @@ import (
 	"text/tabwriter"
 
 	"github.com/cheynewallace/tabby"
-	"gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v2"
 )
 
+type Format struct {
```

Comment:
To make this more generic for future commands, this can be a `FormatOutputId{}`

### Suggestions
This can then have the following methods:

```
(f *FormatOutputId) String()
ToYaml()
ToJson()
```

And have a printing function take an interface that this struct satisfies. Then calling any of the fmt.XXX functions with this object with a `%v` will automatically call `String()`

We can even have it that the caller creates this object that satisfies the interface and passes that into the util package.

Reply:
I have done the modifications as per my understanding of what you asked for. Please review and let me know if this is what you wanted.